src/babel/__tests__/__fixtures__/does-not-override-existing-cache-id/output.js

```diff
@@ -0,0 +1,5 @@
+import { lazyForPaint } from 'react-loosely-lazy';
+const TestComponent = lazyForPaint(() => require('react'), {
+  getCacheId: () => 'react',
```

Comment:
Is it okay that this `getCacheId` doesn't have the require stuff in it?

Reply:
Removed it in this PR since it's no longer needed